argobots.spec

```
@@ -1,22 +1,27 @@
 Name: argobots
-Version: 1.0
-Release: 1%{?dist}
+Version: 1.1~a1
+Release: 1%{?dist}.g7fd1987
 Summary: Lightweight, low-level threading and tasking framework
 Group: System Environment/Libraries
 License: UChicago Argonne, LLC -- Argobots License
 Url: http://www.argobots.org/
 #Source: https://api.github.com/repos/pmodels/$(NAME)/tarball/31703b1
 Source: https://github.com/pmodels/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+# patch to gather all changes from v1.1a1 tag up to ULTs stack unwinding commit
+Patch0: v1.1a1..7fd1987.patch
```

```suggestion
Patch0: v1.1a1...7fd1987.patch
```

To reflect the `git diff` range that created it.

---

Humm, I have the reverse understanding when reading the literature on this topic...

---

This is the `git diff` command you want to use:

```
$ git diff v1.1a1...7fd1987 | sed -e '/^diff --git a\/test\/.gitignore b\/test\/.gitignore/,/^ benchmark\/init_finalize/d' > 
 	$PATH/TO/THIS/PR/v1.1a1...7fd1987.patch
```

Because the diff includes a file (`test/.gitignore`) that is not in the upstream release.  Ideally they should git-ignore that file too.

Note the `...` between the tag and the commit in both the `git diff` command and the output file.  You will need to modify line 11 as suggested above.

---

> This is the `git diff` command you want to use:
> 
> ```
> $ git diff v1.1a1...7fd1987 | sed -e '/^diff --git a\/test\/.gitignore b\/test\/.gitignore/,/^ benchmark\/init_finalize/d' > 
>  	$PATH/TO/THIS/PR/v1.1a1...7fd1987.patch
> ```
> Because the diff includes a file (`test/.gitignore`) that is not in the upstream release. Ideally they should git-ignore that file too.
>
Yes I had finally found it too after manually applying my patch to the Argobots source tarball content. And I have already removed the .gitignore part by editing my patch.

> Note the `...` between the tag and the commit in both the `git diff` command and the output file. You will need to modify line 11 as suggested above.

Again, when reading git-diff man and litterature, I am still convinced that we should use the .. syntax for the commit range. And BTW, both .. and ... versions are identical but my understanding is that this is because both tags/commits are not from diverging branches but in the same one I assume.